README.md

```diff
@@ -21,3 +21,45 @@ Then, whenever you make changes, after it has finished re-compiling, copy the co
 ```
 docker cp dist/. core-web-1:/usr/share/nginx/html/js-plugins/@deephaven/js-plugin-matplotlib/dist/
 ```
+
+## Testing multiple plugins
```

Comment:
I think you can combine this under the previous `Development` header, and then change the previous instructions to just something like:
```
npm install
npm build
```

From the root folder. No need for `docker cp` anymore.